docs/user-guide/deployments/configuration.md

```diff
@@ -550,6 +550,20 @@ In the configuration files of standalone GreptimeDB, the value needs to be set a
 mode = "standalone"
 ```
 
+### Heartbeat configuration
+
```

Comment:
```suggestion
Heartbeat configuration is available in `frontend`, `datanode` and `standalone`
```

Reply:
"standalone" doesn't need heartbeat

Reply:
@MichaelScofield I modified the suggestion comments, PTAL